Software/bsw/static/COM/CanIf/src/CanIf.c

```diff
@@ -174,35 +174,35 @@ void CanIf_TxConfirmation(PduIdType CanTxPduId)
 			PduUser = CanIf_ConfigPtr->CanIfInitCfgRef->CanIfTxPduCfgRef[CanTxPduId].CanIfTxPduUserTxConfirmationUL;
 			if(CAN_NM == PduUser)
```

Comment:
CAN_NM_TX_CONFIRMATION as we agreed (as the same name implemented in the PBcfg file) not CAN_NM to be more specific